packages/staking/src/governance/api.ts

```diff
@@ -0,0 +1,62 @@
+import {AxiosInstance} from 'axios'
+import {axiosClient, GOVERNANCE_ENDPOINTS} from './config'
+
+export type GovernanceApi = {
+  getDRepById: (drepId: string) => Promise<{txId: string}>
+  isRegisteredDRep: (drepId: string) => Promise<boolean>
+}
+
+export const initApi = ({
```

Comment:
`governanceApiMaker`

Reply:
if only for governance, if the module will expose everything staking later, `stakingApiMaker`